tasks/managed/prepare-data-acceptable-bundles/tests/test-prepare-data-acceptable-bundles.yaml

```diff
@@ -0,0 +1,270 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-prepare-data-acceptable-bundles
+spec:
+  description: |
+    Run the prepare-data-acceptable-bundles task and verify
+    that it correctly modifies the snapshot to include data-acceptable-bundles
+    repositories as components marked as public.
```

Comment:
Hmm, shouldn't it honor the `public` setting of the component? Your input data does not specify `public` at all, so I assume it will always set it to public. Is that correct?

Reply:
ok, good